tests/pytests/test_geo.py

```diff
@@ -18,13 +18,29 @@ def testGeoFtAdd(env):
   env.expect('FT.SEARCH', 'idx', '@g:[1.23 4.56 1 km]').equal([1L, 'geo2', ['g', '1.23,4.56']])
 
 def testGeoDistanceSimple(env):
+  env.skipOnCluster()
   env.expect('ft.create', 'idx', 'schema', 'name', 'text', 'location', 'geo', 'hq', 'geo').ok()
   env.expect('FT.ADD', 'idx', 'geo1', '1', 'FIELDS', 'location', '1.22,4.56', 'hq', '1.25,4.5').ok()
   env.expect('FT.ADD', 'idx', 'geo2', '1', 'FIELDS', 'location', '1.24,4.56', 'hq', '1.25,4.5').ok()
   env.expect('FT.ADD', 'idx', 'geo3', '1', 'FIELDS', 'location', '1.23,4.55', 'hq', '1.25,4.5').ok()
   env.expect('FT.ADD', 'idx', 'geo4', '1', 'FIELDS', 'location', '1.23,4.57', 'hq', '1.25,4.5').ok()
   env.expect('FT.SEARCH', 'idx', '@location:[1.23 4.56 10 km]', 'nocontent').equal([4L, 'geo1', 'geo2', 'geo3', 'geo4'])
 
+  # test profile
+  env.cmd('FT.CONFIG', 'SET', '_PRINT_PROFILE_CLOCK', 'false')
+  res = [4L, 'geo1', 'geo2', 'geo3', 'geo4',
+             ['Total profile time'],
```

Comment:
indentation

Reply:
This one is OK
The indentations mimick the different array levels